test/web-auth/web-auth.test.js

```diff
@@ -1487,7 +1515,7 @@ describe('auth0.WebAuth', function() {
       stub(SilentAuthenticationHandler.prototype, 'login', function(usePostMessage, cb) {
```

Comment:
Please clarify in the test name why this token validation is going to fail and which token you're talking about (I'm guessing the AT)

Reply:
I renamed the test, but I only changed the hash value it doesn't try to validate the access_token